### ISSUE TITLE
Convert JS ES6 code to use async/await functions

### ISSUE THREAD

**Issue by SeannyPhoenix** [postmanlabs/postman-code-generators#302]
**Is your feature request related to a problem? Please describe.**
No problem, just updating code.

**Describe the solution you'd like**
I would like to convert functions to use async/await functions.

Example existing code output for nodejs-axios:
```javascript
const axios = require('axios');
let data = JSON.stringify({"field":"value"});

let config = {
  method: 'post',
  url: 'https://example.com/api/v1/endpoint',
  headers: { 
    'Authorization': 'Bearer auth_token', 
    'Content-Type': 'application/json'
  },
  data : data
};

axios(config)
.then((response) => {
  console.log(JSON.stringify(response.data));
})
.catch((error) => {
  console.log(error);
});
```

Proposed output:
```javascript
const axios = require('axios');
let data = JSON.stringify({"field":"value"});

const config = {
  method: 'post',
  url: 'https://example.com/api/v1/endpoint',
   headers: {
   'Authorization': 'Bearer auth_token',
   'Content-Type': 'application/json'
  },
  data: data
};

async function makeCall(config){
  try {
    const response = await axios(config);
    console.log(JSON.stringify(response.data));
  } catch (error) {
    console.log(error);
  }
}

makeCall(config);
```

**Comment by zeel91297** [comment]
I would like to work on this feature.

**Comment by dhwaneetbhatt** [comment]
This is fixed in the latest version v10.12. Closing this issue.
